install-and-configure/install/cloud-integration/oracle/usage-api-cloud-cost-integration.md

```diff
@@ -0,0 +1,123 @@
+# Oracle Usage API Cloud Cost Integration
+
+This document describes how to integrate Kubecost with the Oracle Usage API to build Cloud Costs for your tenancy. This will largely be done via the OCI CLI which you will need admin privileges on.
+
+### Before Starting
+
+Ensure that you have the oracle CLI installed and configured for your account. Documentation on this can be found [here](https://docs.oracle.com/en-us/iaas/Content/API/SDKDocs/cliinstall.htm).
+
+You will also need the tenancy ID and region of the tenancy in which your clusters are running. This can be found in the Oracle Cloud console by selecting "Tenancy: <TENANCY-NAME>" from the profile drop down in the top right corner. Under "Tenancy information" the tenancy ID is labeled "OCID" and the region is marked "Home region". The region name listed here will need to be changed to a region identify found [here](https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm).
+
+### Authentication and Authorization
+
+Access to the Usage API is defined in a policy that applies to a group. Using credentials for a User that is a member of that group Kubecost can gain access to the API.
+
+
+#### Creating a User
+
+Run the following command to create a user. You will need to provide an email address for this user which does not have to be unique among users. Be sure to save the values mentioned below.
+
+```sh
+oci iam user create \
+--name kubecostUser \
+--description="Access point for kubecost" \
+--email <REQUIRED_EMAIL>
+```
+
+Save the user ID which can be found in the "id" property of the output.
+
+
+Save the compartment ID found in the "compartment-id" property of the output.
+
+
+#### Creating a Group
+
+Next create a group which will have the policy attached to it.
+
+```sh
+oci iam group create \
+--name=kubecost \
+--description="group for kubecost"
+```
+
+Save the group ID found in the "id" property of the output.
+
+
+Add user to the new group.
+
+```sh
+oci iam group add-user \
+--group-id $OCI_GROUP_ID \
+--user-id $OCI_USER_ID
+```
+
+Create a policy for group.
+
+```sh
+oci iam policy create \
+--compartment-id $OCI_COMPARTMENT_ID \
+--name kubecostUserPolicy \
+--description="policy for kubecost" \
+--statements='["ALLOW GROUP kubecost to read all-resources IN TENANCY"]'
+```
+
+
+#### Create and add an API key for the User
+
+Start by generating a set of RSA PEM files.
+> If following this guide, be sure to change the name of the pem files you are creating. Failing to do can cause the CLI to stop working
+https://docs.oracle.com/en-us/iaas/Content/API/Concepts/apisigningkey.htm#apisigningkey_topic_How_to_Generate_an_API_Signing_Key_Mac_Linux
+
+
+Upload the public key to the user you created in the last step.
+
+```sh
+oci iam user api-key upload \
+--user-id $OCI_USER_ID 
+--key-file="/path/to/key_public.pem"
+
```

Comment:
```suggestion
```sh
oci iam user api-key upload \
--user-id $OCI_USER_ID 
--key-file="/path/to/key_public.pem"
```

```